### ISSUE TITLE
styleguide vs ABAP Programming guidelines

### ISSUE THREAD

**Issue by larshp** [SAP/styleguides#253]
there is https://help.sap.com/doc/abapdocu_752_index_htm/7.52/en-US/abenabap_pgl.htm and other places in the ABAP help, which defines guidelines/recommendations

Personally I think it would be nice to have everything in one place

However, there is also a possibility of duplicating information

Any thoughts?

**Comment by preinboth** [comment]
I would be in favor of everything in one place, and this on github.

**Comment by sautermi0** [comment]
I think the intentions of CleanABAP and the SAP Help are quite different... CleanABAP is a community-driven guide on how-to apply CleanCode to ABAP. The SAP Help is a description from a central point of view, which is pretty close to the ABAP language development... So, in my humble opinion, it's okay to have two places. One for the community-driven guide and one for the perspective of the people who develop(ed) the language.

**Comment by larshp** [comment]
"community driven", well...

https://blogs.sap.com/2023/01/26/clean-code-community-contribution-discussion-review-and-decision-process/

"Clean ABAP review and discussion moderators meet every second week", "Once every quarter, the agile software engineering circle", "Decision Board for ABAP", this sounds much like it SAP driven

from an outside perspective, I've not found it possible to identify who the owners are. And there is no link to the blog post, or description of the SAP process in the repository

**Comment by ConjuringCoffee** [comment]
> this sounds much like it SAP driven

I agree. At the moment, Clean ABAP is more "community supported", with SAP still being the owner of the styleguide. I don't think anything will be added if they don't agree with it.
